### DISCUSSION TITLE
HD 200 sometimes sluggish response sometimes has trouble connecting

### DISCUSSION THREAD

**Discussion by LoisGNS** [OpenSageTV/Community#176] [SageTV Extenders (HD300,HD200)]
The HD 200 I've been using since 2009-ish has developed some issues: 

From time to time it gets unresponsive to the remote for a while, then eventually snaps out of it. That _could_ by my server running short of memory or Java heap, but hard to tell. Sometimes power-cycling the HD 200 fixes it.

Lately, when power-cycling or awaking from standby it doesn't see my server. Last night I had to power-cycle several times before it would connect properly. I checked the server and it was up & running just fine.

Could this be a hardware issue? I'd think if it were firmware, the issues would be constant rather than intermittent. Could it even be a simple as a bad or iffy ethernet cable or connection?

Or, is there a component in there that might need to be replaced? My husband has done component-level repairs on ancient computers, but isn't sure what to look for here. Fortunately, I have some spare HD 200s and HD 300s that I could sub in, but I prefer to keep everything working as long as possible since they aren't making any more of them.

**Comment by wayner9** [comment]
I haven't  had issues with these and I have had a ton of them.  I think it has been more reliable than the HD300 which overheats sometimes.

I have seen issues when you need to reboot several times, especially if you don't have a valid HDMI connection.  Sometimes it boots up better if you disconnect the ethernet cable, boot up, and then plug in the ethernet cable.

**Comment by JustFred51** [comment]
The most likely hardware problem I would suspect would be the power supply (wall wart).  Try swapping it with another from your other HD200s.

**Comment by LoisGNS** [comment]
Thanks. Yes, last night I had to power-cycle several times before it worked correctly. I'll try swapping out the ethernet cable, hdmi cable, and wall wart one at a time and see if the problem recurs.

Edit: Swapped the ethernet cable. We also checked the wall wart for voltage. It is labeled as 12v but tested around 6v, and a spare tested at 7v. Another one in use in another room tested at 10 or 11v, but I'd rather leave it intact for use in the other room. So that's interesting. Husband is currently scouring his collection of wall warts to see if there's a suitable replacement (or two) among them. Then we'll have to see how it behaves when we use it a little later tonight.

**Comment by TGsell1** [comment]
Last ditch… try plugging HDMI into a monitor not a TV they have different signal communications Sent from my iPhoneOn Dec 6, 2024, at 10:17 PM, LoisGNS ***@***.***> wrote:﻿
Thanks. Yes, last night I had to power-cycle several times before it worked correctly. I'll try swapping out the ethernet cable, hdmi cable, and wall wart one at a time and see if the problem recurs.
